app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -853,6 +853,21 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(CTA_CLICKED.eventName, props)
     }
 
+    /**
+     * Sends data associated with the search results page viewed to segment.
+     * @param discoveryParams: DiscoveryParams
+     * @param count: Int
+     * @param sort: DiscoveryParams.Sort
+     */
+    fun trackSearchResultPageViewed(discoveryParams: DiscoveryParams, count: Int, sort: DiscoveryParams.Sort) {
+        val props = AnalyticEventsUtils.discoveryParamsProperties(discoveryParams).toMutableMap()
+        props[CONTEXT_PAGE.contextName] = SEARCH.contextName
+        props[DISCOVER_SORT.contextName] = sort.name.toLowerCase(Locale.ROOT)
```

Comment:
So if you look at the discoveryParamsProperties method, it is handling adding the `discover_sort` property automatically. You can either pass it in explicitly, or it will default to whatever value is in `discoveryParams.sort`. If you include this property in this method, it will be sending `discover_sort` twice.